src/content/basics/app-catalog/index.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "The Giant Swarm App Catalog"
+description = "Overview of the Giant Swarm App Catalog, how it works and what to expect."
+date = "2019-02-11"
+weight = 15
+type = "page"
+categories = ["basics"]
++++
+
+# The Giant Swarm App Catalog
+
+The _Giant Swarm App Catalog_ refers to a set of features and concepts that allow
+you to browse, install and manage the configurations of apps (such as prometheus)
+from a single place; the Control Plane.
+
+Through this we are also providing you a list of curated _Managed Apps_.
+In the near future you might also see _Apps_ provided by our partners.
+
+These collections of _Apps_ are grouped in _App Catalogs_, which in turn are
+browseable through our web interface.
+
+So to clarify a bit: the _Giant Swarm App Catalog_ refers to the whole feature,
+and an _App Catalog_ is a collection of _Apps_.
+
+We'll provide some _App Catalogs_, but so will our partners, and you too can set
+up your own internal catalog(s) for your whole company to use.
+
+### What makes up the Giant Swarm App Catalog?
+
+Technically the Giant Swarm App Catalog is implemented as a set of operators
+running on your Control Plane and tenant clusters. These operators watch various
+Custom Resources, some created by us, and others created by you, which make up
+the desired state of the Giant Swarm App Catalog.
+
+For example, this "App" Custom Resource indicates that you want Instana installed
```

Comment:
Minor but is there a reason we are using Instana here? I know they are friends but a more common example from community woud be better IMO.

Reply:
I'm happy to pick something else. Any suggestion?

Reply:
What about grafana? Something that is a bit more generic.